PFS-IPMU-00001/infra.rst

```diff
@@ -0,0 +1,85 @@
+Infrastructure overview (storage, VM host)
+======
+
+ToC
+
+* `Storage service configuration`_
+* `VM host configuration`_
+
+Storage service configuration
+------
+
+Two categories of storage services are provided, one is actively used service 
+backend, and another is archive and/or backup service backend. 
+
+For actively used service backend, storage is exposed through VM guest which 
+mounts physical drive through virtio. 
+Disk drives are placed in one SAS expander box and controlled by RAID card at 
+host computer as RAID1 with hot spare. On hardware issue, it is possible to 
+move disk drives into another host computer to continue services. 
+To survive UPS failure, both SAS expander box and host computer shall be 
+connected to two separated UPS as redundant power source. 
+Four RAID1 are mounted and exposed by three VM guests: 
+
+* pd (pfsdisk): /virt and /server
+* pc (pfscalc): /data1
+* sd (simdata): /simdata
+
```

Comment:
VM guest names are pd2, pc2 and sd2 now??

Reply:
yes. need to add that the last digit is a revision number of instance...